lib/utils/form.ts

```diff
@@ -52,7 +52,7 @@ export function checkEligible(
   };
 
   if (isMainApplicant) {
-    if (!isOver18(applicant)) {
+    if (!applicantEqualToOrOlderThanAge(applicant, 18)) {
```

Comment:
I think this would be better kept as a descriptive function, which has the value `18` set in one place rather than in multiple files.